src/crypto/crypto_ec.cc

```diff
@@ -706,7 +706,44 @@ WebCryptoKeyExportStatus ECKeyExportTraits::DoExport(
     case kWebCryptoKeyFormatSPKI:
       if (key_data->GetKeyType() != kKeyTypePublic)
         return WebCryptoKeyExportStatus::INVALID_KEY_TYPE;
-      return PKEY_SPKI_Export(key_data.get(), out);
```

Comment:
This routine is also used for Ed25519 and the like, so `PKEY_SPKI_Export(key_data.get(), out);` should still be used if `EVP_PKEY_id(pkey.get())` is one of `EVP_PKEY_ED25519`, `EVP_PKEY_ED448`, `EVP_PKEY_X25519`, or `EVP_PKEY_X448`.